DigitalLearningSolutions.Web/Controllers/NotificationPreferencesController.cs

```diff
@@ -13,6 +13,7 @@ namespace DigitalLearningSolutions.Web.Controllers
     using Microsoft.Extensions.Logging;
 
     [ValidateAllowedApplicationType]
+    [SetApplicationTypeAndSelectedTab(nameof(ApplicationType.Main), nameof(Tab.MyAccount))]

```

Comment:
This can be on different applications as well - anything on the MyAccount tab can.